HACKING.md

```diff
@@ -509,6 +509,22 @@ To create it use:
 oc create secret generic splunk-casc-cfg \
     --from-file=splunk.yaml
 ```
+### [OPTIONAL] Create brew secrets
+
+Brew requires a couple of secrets in order to work. All of them are stored in Bitwarden.
+We need the certificate (brew-ca), the keytab (brew-keytab), the koji configuration (koji-conf) and the kerberos configuration (krb5-conf). Make sure to meet the secret names, these names are used in the brew job.
+
+To create it use:
+```
+oc create secret generic brew-ca \
+    --from-literal=filename=ca.crt --from-file=data=ca.crt
+oc create secret generic brew-keytab \
+    --from-literal=filename=brew-keytab --from-file=data=brew-keytab
+oc create secret generic koji-conf \
+    --from-literal=filename=koji-conf --from-file=data=koji-conf
+oc create secret generic krb5-conf \
+    --from-literal=filename=krb5-conf --from-file=data=krb5-conf
```

Comment:
Hmm, these look off. It's creating a secret with keys `filename` and `data`, but I think based on the code below, the key names should be e.g. `ca.crt`, `koji.conf`, `krb5.conf`, no?